examples/sizes.txt

```diff
@@ -61,5 +62,5 @@ tuple_support                        438     290     290
 unary                                130     92      92      
 undefined_phi_args/baddie            346     346     311     
 unssa                                442     371     371     
-voting/voting                        1605    1505    1505    
+voting/voting                        1624    1523    1523    
```

Comment:
Why the increase?

Reply:
One of the subroutines takes a mutable type as an argument.
It's never used as such, the detection of whether to generate the implicit returns is simply "is this method called from anywhere expect an ARC4 Router" is MVP and can definitely be improved.
But it's a small price to pay for the ability to have mutable arguments that function as expected.